### DISCUSSION TITLE
mTLS for HTTPS health checks

### DISCUSSION THREAD

**Discussion by Slepwin** [loxilb-io/loxilb#234] [Ideas]
Hi,

Please add to your roadmap mutual TLS support for HTTPS health-checks, thanks!

**Comment by nik-netlox** [comment]
Yes, we are on it! We will role this feature out soon.

**Comment by nik-netlox** [comment]
Hi @Slepwin,
mTLS support has been added. Currently, it is available in beta. You may have a look.
